app/importers/est_data.rb

```diff
@@ -0,0 +1,62 @@
+class EstData
+  include Importer
+  LOGIN_URL = 'https://admin.export.gov/site_login'
+  ENDPOINT = 'https://admin.export.gov/admin/envirotech_solutions.json'
+
+  cattr_accessor :web_auth, :username, :password
+
+  COLUMN_HASH = {
```

Comment:
We don't have any guidance on this so I suggest indexing all available fields, snake_casing their names. If that's what you've done, then that's enough.